src/Swashbuckle.AspNetCore.Swagger/ISwaggerDocumentSerializer.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Microsoft.OpenApi;
+using Microsoft.OpenApi.Models;
+using Microsoft.OpenApi.Writers;
+
+namespace Swashbuckle.AspNetCore.Swagger
+{
+    /// <summary>
+    /// Provide an implementation for this interface if you wish to customize how the open api document is exactly written.
+    /// SerializeDocument will be called in that case, instead of the Microsoft built in Serialize methods.
```

Comment:
```suggestion
```

Reply:
I might be going blind, but what exactly is different in this suggestion?

Reply:
🤦‍♂️